setup.py

```diff
@@ -11,14 +11,14 @@
 version = loads(package_json).get('version')
 
 setup(
-  author='camargo',
-  author_email='Christopher.A.Camargo@jpl.nasa.gov',
+  author='cmak',
+  author_email='carter.mak@jpl.nasa.gov',
   description='Standardized JSON Schema for authoring multi-mission sequences',
   include_package_data=True,
   long_description=long_description,
   long_description_content_type='text/markdown',
   name='seq-json-schema',
-  packages=['seq-json-schema'],
+  packages=['jsonschema', 'seq-json-schema'],
```

Comment:
If this is here as a dependency, it should be entered as `requires` -- `packages` is for what the distribution provides

Reply:
Done!